src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -432,6 +432,17 @@ public String getQuestionResultStatisticsCsv(
         return csv;
     }
     
+    private List<String> getTeamNames(FeedbackSessionResultsBundle bundle) {
+        List<String> teamNames = new ArrayList<String>();
+        for (String studentEmail : bundle.emailTeamNameTable.keySet()) {
+            String teamName = bundle.emailTeamNameTable.get(studentEmail);
```

Comment:
Looks like you want to iterate over <code>bundle.emailTeamNameTable.values()</code> here